.npmignore

```diff
@@ -1,12 +1,10 @@
 CONTRIBUTING.md
 Makefile
-SPONSORS.md
-docs/
 examples/
 reports/
 test/
-var/
```

Comment:
As above re: `var/`.

Reply:
Also, I presume you want `templates` added here too, so only included with Git but not npm (caught this looking at `npm publish --dry-run`)...

Reply:
And adding `.DS_Store` wouldn't hurt if publishing from a Mac...